src/Form/FormGroup.js

```diff
@@ -1,6 +1,36 @@
 import React from 'react';
 import Box from '../Box';
+import styled, { css } from 'styled-components';
+import { createComponent } from '../utils';
 
-const FormGroup = ({ children, padding = 3 }) => <Box mb={padding}>{children}</Box>;
+const Legend = createComponent({
+  name: 'Legend',
+  tag: 'legend',
+  style: ({
+    theme,
+    color = theme.colors.primary,
+  }) => css`
+    font-weight: 700;
+    margin-bottom: 1rem;
+    font-size: 10px;
+    text-transform: uppercase;
+    letter-spacing: 0.25px;
+    color: ${color};
+  `,
+});
+
+const Container = styled(Box)`
+  * + & {
+    margin-top: 1.5rem;
+  }
+`;
+
+const FormGroup = ({ legend, children, padding = 3 }) => (
```

Comment:
To keep with underlying html semantics, let's call this fieldset

Reply:
thats gonna require changing all the FormGroup instances in the emr and web app - we can export it under both names for now?